### PR TITLE
fix(react-vapor): bump rc-tooltip, rc-slider, react-bootstrap

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Bumping major versions of those dependencies in order to avoid having core-js 2.6 in the esm bundle:
- rc-slider
- rc-tooltip
- react-bootstrap

Also bumped all rollup plugins and cleaned up the config so that we no longer have warnings:
image

### Potential Breaking Changes

BREAKING CHANGE: 
- Tooltip is now placed on top if placement not specified
- Tooltip props might have changed too because of the react-bootstrap major version bump (hard to tell)

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/package.json [hunk 1]
```diff
@@ -36,12 +36,15 @@
     },
     "dependencies": {
         "@types/codemirror": "0.0.109",
-        "@types/rc-slider": "8.6.6",
         "chosen-js": "1.8.7",
+        "classnames": "2.3.1",
+        "coveo-styleguide": "^28.2.1",
+        "dnd-core": "2.6.0",
+        "moment": "2.29.1",
```
Comment (gdostie) [inline packages/react-vapor/package.json:43]: Moved in deps from devDeps

File: packages/react-vapor/src/components/select/tests/SingleSelectConnected.spec.tsx [hunk 2]
```diff
@@ -169,9 +169,12 @@ describe('Select', () => {
...
             expect(screen.queryByRole('button', {name: /select one/})).not.toBeInTheDocument();
 
-            userEvent.click(screen.queryByRole('img', {name: /clear icon/}));
+            act(() => {
+                userEvent.click(screen.getByRole('img', {name: /clear icon/}));
+            });
 
-            expect(screen.getByRole('button', {name: /select one/})).toBeVisible();
+            screen.logTestingPlaygroundURL();
+            // expect(screen.getByRole('button', {name: /select one/})).toBeVisible();
```
Comment (gdostie) [inline packages/react-vapor/src/components/select/tests/SingleSelectConnected.spec.tsx:177]: Woops forgot this one.. on it

**Review by aiheon — APPROVED** [review]
GG :tada:

**Review by kdang-coveo — APPROVED** [review]

File: packages/react-vapor/rollup.config.js [hunk 3]
```diff
@@ -104,38 +87,41 @@ export default {
...
+            d3: 'd3',
+            jquery: '$',
+            react: 'React',
+            'react-dom': 'ReactDOM',
+            'react-redux': 'ReactRedux',
+            redux: 'Redux',
+            underscore: '_',
+            'coveo-styleguide': 'VaporSVG',
+            'underscore.string': 's',
+            'react-dom/server': 'ReactDOMServer',
```
Comment (toofff) [inline packages/react-vapor/rollup.config.js:101]: Duplicate code, create a constant would surely be better?

File: postcss.config.js [hunk 4]
```diff
@@ -1,16 +0,0 @@
...
-                'last 3 Chrome versions',
-                'last 3 Firefox versions',
-                'Explorer >= 11',
-                'last 2 Edge versions',
-                'last 3 Opera versions',
-                'Safari >= 8',
-            ],
-        }),
-    ],
-};
```
Comment (toofff) [inline postcss.config.js:16]: Why remove this configuration?

Comment (gdostie) [reply postcss.config.js:16]: The `browsers` option of the autoprefixer is deprecated, it now automatically uses the `browserslist` config we have in the [package.json](https://github.com/coveo/react-vapor/blob/master/package.json#L34-L39)

**Pull request merged**